src/validators.js

```diff
@@ -0,0 +1,8 @@
+export class Validators {
```

Comment:
Any technical reason for a class instead of function? 

If you use a class with static, would it make sens to freeze it?

Reply:
Zero technical reason.
I've been inspired by Angular validators and wanted to have `Validators.required` syntax ^^
I agree a freeze would be nice, do you think a class with static getters is a wrong choice or it's ok and I just freeze it ?

Reply:
It depend, I would use standalone functions (feel like using class is kinda an usage abuse).

Reply:
I go for it then 👍